README.md

```diff
@@ -239,4 +239,4 @@ This is not an officially supported Google product.
 [goth]: https://hex.pm/packages/goth
 [elixir-samples]: https://github.com/GoogleCloudPlatform/elixir-samples
 [oauth2-library]: https://github.com/scrogson/oauth2
-[auth-sample]: https://github.com/GoogleCloudPlatform/elixir-samples/tree/master/auth
\ No newline at end of file
+[auth-sample]: https://github.com/GoogleCloudPlatform/elixir-samples/tree/master/auth
```

Comment:
What changed in this line?

Reply:
Nothing; i don't know why it's marked as changed

Reply:
Looks like it might be the addition of the trailing file newline. Not sure why the github UI doesn't show it.